src/python/pants/engine/build_files.py

```diff
@@ -26,232 +33,284 @@
 
 
 class ResolvedTypeMismatchError(ResolveError):
-  """Indicates a resolved object was not of the expected type."""
+    """Indicates a resolved object was not of the expected type."""
 
 
 def _key_func(entry):
-  key, value = entry
-  return key
+    key, value = entry
+    return key
 
 
```

Comment:
Unfortunately, this file seems to have been formatted with `black` =(

Reply:
Skipping review of this because the diff is unclear to me.

Reply:
Hmm I'm not sure what happened here, I thought I'd reverted my attempts to apply black. Will fix it up.